alembic/versions/3ae765d6776d_migrate_cgstats_data.py

```diff
@@ -0,0 +1,53 @@
+"""Migration of data in the cgstats unaligned database table into the cg SampleLaneSequencingMetrics table.
+
+Revision ID: 3ae765d6776d
+Revises: 367ed257e4ee
+Create Date: 2023-06-22 15:18:18.623714
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "3ae765d6776d"
+down_revision = "367ed257e4ee"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Execute the query and insert data into the existing table
+    op.execute(
+        """
+        INSERT INTO cg.sample_lane_sequencing_metrics (
+            sample_internal_id,
+            flow_cell_name,
+            flow_cell_lane_number,
+            sample_total_reads_in_lane,
+            sample_base_fraction_passing_q30,
+            sample_base_mean_quality_score,
+            created_at
+        )
+        SELECT
+            cgstats.sample.limsid AS sample_internal_id,
+            cgstats.flowcell.flowcellname AS flow_cell_name,
+            cgstats.unaligned.lane AS flow_cell_lane_number,
+            cgstats.unaligned.readcounts AS sample_total_reads_in_lane,
+            cgstats.unaligned.q30_bases_pct AS sample_base_fraction_passing_q30,
```

Comment:
This field is reported as a fraction and not a percentage in our new table - do we want to transform the data we are importing to a fraction as well?

Reply:
Ok, I think so to! I think it is nice to keep everything related to the migration in a single revision, but we will likely need to revise it anyway